app/views/quotes/_quote.html.erb

```diff
@@ -1,5 +1,5 @@
 <div>
-  <%= quote.quoted_person %> said, <q><%= quote.body %></q>, in the <%= quote.location %>
+  <%= quote.quoted_person %> said, <q><%= quote.body %></q>, in <%= quote.location %>.
```

Comment:
This isn't part of adding an example quote. This is a _real_ quote :stuck_out_tongue:. Should be own commit.